### PR TITLE
Edu-Homework component

### PR DESCRIPTION
**Pull request by Osman-Hajr**
A component for the EduHomework is created. 
A drop down menu to select a module to give score.
A drop down menu to select the week of the module to give score.
Scores to choose from to rate the module and the week with confirmation alert before final submission.

### CODE REVIEW

File: src/pages/EduHomework/EduHomework.jsx [hunk 1]
```diff
@@ -3,14 +3,15 @@ import './eduhomework.scss';
...
-
+import EduHomeworkComponent from '../../components/EduHomeworkComponent/EduHomeworkComponent';
 
 export default function EduHomework() {
 
     return (
         <div className='eduHomeworkPage'>
             <HeadingPageComponent title={'Edu Homework'} />
             <Link className='eduHomeworkPage__btn' to='/student-tracker'>&larr; goBack</Link>
+            <EduHomeworkComponent />
```
Comment (Denis-Andrei) [inline src/pages/EduHomework/EduHomework.jsx:14]: This component shouldn't be included here. EduHomework Component is for the /students/:profile page(which is not created yet).

**Review by Denis-Andrei — CHANGES_REQUESTED** [review]

File: src/pages/EduHomework/EduHomework.jsx (hunk 1 above)
Comment (Osman-Hajr) [reply src/pages/EduHomework/EduHomework.jsx:14]: Changes made accordingly.

Comment (ISTANBULBEKLE) [reply src/pages/EduHomework/EduHomework.jsx:14]: Good, job guys, fantastic.

**Review by ISTANBULBEKLE — APPROVED** [review]
Great job guys, thanks alot.

**Review by Denis-Andrei — APPROVED** [review]
Looks good

**Pull request merged**